sources/Debate-frontend/src/app/services/auth.service.ts

```diff
@@ -0,0 +1,44 @@
+import { Injectable } from '@angular/core';
+import { UserLogIn } from '../dto/user-log-in';
+import { UserDto } from '../dto/user.dto';
+import { Observable } from 'rxjs/internal/Observable';
+import { HttpClient } from '@angular/common/http';
+import { CookieService } from 'ngx-cookie-service';
+import { map } from 'rxjs/operators';
+import { BehaviorSubject } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AuthService {
+  private endpoint = 'http://localhost:8080/api';
```

Comment:
For the future - we have to move backend URL to environment.properties.ts and create two properties files - one for development and one for production.

Reply:
This doesn't have to be done in this pull request - we can create the properties files later, but we have to be aware of this issue

Reply:
but currently we use private fields all services so it's a task to do in the future